snapcraft/plugins/python2.py

```diff
@@ -40,182 +40,35 @@
       A list of dependencies to get from PyPi
 """
 
-import glob
 import logging
 import os
-import re
 
-import snapcraft
-from snapcraft import common
+from snapcraft.plugins import python3
 
 logger = logging.getLogger(__name__)
 
 
-class Python2Plugin(snapcraft.BasePlugin):
+class Python2Plugin(python3.Python3Plugin):
```

Comment:
El 01/09/16 a las 11:04, Leo Arias escribió:
>
> In snapcraft/plugins/python2.py 
> <https://github.com/snapcore/snapcraft/pull/771#discussion_r77179576>:
>
> >
> >  logger = logging.getLogger(__name__)
> >
> >
> > -class Python2Plugin(snapcraft.BasePlugin):
> > +class Python2Plugin(python3.Python3Plugin):
>
> neither. In a common module.
>

I don't know, this gives the plugins we write special treatment, this 
module would have to be a plugin as well and we risk having a useless 
plugin like the jdk one.